payments/index.ts

```diff
@@ -0,0 +1,80 @@
+import { TransactionInput } from "./types"
+import Stripe from "stripe"
+import mongoDB, { MongoClient } from "mongodb"
+
+import * as dotenv from "dotenv"
+dotenv.config()
+import express from "express"
+
+const app = express()
+const port = 5000
+const stripe = new Stripe(process.env.STRIPE_SECRET_KEY, {
+	apiVersion: "2022-11-15",
+	typescript: true,
+})
+const collections: { paymentintents?: mongoDB.Collection } = {}
+
+// function to connect to MongoDB
+async function connectToDatabase() {
```

Comment:
this function is a little misleading, it creates the database, connects to it and defines a new collection, "paymentintent"

- [ ] break up the function to connect (returns the db client) & add new collection